packages/frontend/src/components/accounts/recovery_setup/RecoveryOption.js

```diff
@@ -193,7 +193,7 @@ const Icon = ({option, color}) => {
         case 'ledger':
             return <HardwareWalletIcon color={color}/>;
         default:
-            return;
+            return '';
```

Comment:
Just curious about why we had to change this to return a string, and are we sure that this doesn't have any unintended impact on existing UI flows?

Reply:
Yep it does not have any impact on existing UI and was only necessary to make `<RecoveryOption/>` support a version without a provided icon:

<img width="308" alt="Screen Shot 2022-02-10 at 4 58 58 PM" src="https://user-images.githubusercontent.com/24921205/153522072-07c22631-14c5-4bfe-bc62-d0f9e286411f.png">